dir.props

```diff
@@ -71,6 +54,11 @@
   <!-- Import Build tools common props file where repo-independent properties are found -->
   <Import Project="$(ToolsDir)Build.Common.props" Condition="Exists('$(ToolsDir)Build.Common.props')" />
 
+  <PropertyGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(OSEnvironment)</InputOSGroup>
+    <BuildToolsTargets45 Condition="'$(OsEnvironment)'=='Windows_NT'">true</BuildToolsTargets45>
```

Comment:
@ericstj mentioned a few more places over in https://github.com/dotnet/buildtools/issues/1284 that I haven't touched yet, so I am keeping it defined for now to be safe.